lib/platform/node/index.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   classes: {
     URLSearchParams: require('./classes/URLSearchParams'),
     FormData: require('./classes/FormData'),
-    Blob: typeof Blob !== 'undefined' && Blob || null
+    Blob: Blob !== undefined ? Blob : null
```

Comment:
This is not an equivalent code. 
1) Blob !== undefined will raise a ReferenceError on strict mode;
2) Axios is cross-platform and it would be better to treat all falsies as null since legacy versions of nodes can have user-defined Blob variables that can be anything other than a function or undefined;

Reply:
hmm i was really unsure about this code and thought about it. Legacy versions of nodes can cause problems